pkg/utils/accelerators/tpu.go

```diff
@@ -211,6 +216,10 @@ func AddTPUVariables(pod *corev1.Pod, size int) error {
 			Name:  TpuWorkerId,
 			Value: fmt.Sprint(tpuWorkerId),
 		},
+		corev1.EnvVar{
+			Name:  TpuName,
+			Value: fmt.Sprint(pod.Name),
```

Comment:
the tpu_name here should be the identifier for a tpu slice. So for multi-host tpu slice, all hosts share the same tpu_name value. Let's use leader pod's name here.

Reply:
actually this may cause the ray only to use one host to load the model which OOM happens on that host?

Reply:
Updated to use leader pod's name